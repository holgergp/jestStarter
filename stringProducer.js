export const stringProducer = a => {
  return "myString" + a;
};
