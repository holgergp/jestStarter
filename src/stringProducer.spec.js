import {stringProducer} from './stringProducer';

describe("stringProducer", () => {
  it("should prepend the given string", () => {
    const myString = stringProducer("Reactjs");
    expect(myString).toMatchSnapshot();
  });
});


