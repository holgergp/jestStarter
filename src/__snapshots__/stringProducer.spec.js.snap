// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`stringProducer should prepend the given string 1`] = `"myStringReactjs"`;
